src/components/menuNavbar/MenuNavbar.tsx

```diff
@@ -0,0 +1,45 @@
+import { useState } from 'react';
+import { ItAcademyLogo } from '../../assets/svg';
+
+const MenuNavbar = () => {
+  const [activeItem, setActiveItem] = useState('Perfiles');
+  const menuItems = [
```

Comment:
Don't declare constants within the component, nor in the same file. The component should render props that receives in order to be reusable.